src/index.html

```diff
@@ -11,12 +12,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <!-- #endregion HEAD-->
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html">
+          <img
+            src="/src/images/logo.png"
```

Comment:
Consider using a relative path for the `src` attribute of the logo image to avoid issues if the file structure changes or if the site is deployed to a different environment.